DSharpPlus.VoiceNext/DSharpPlus.VoiceNext.csproj

```diff
@@ -1,23 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
-    <PropertyGroup>
-        <AssemblyName>DSharpPlus.VoiceNext</AssemblyName>
-        <RootNamespace>DSharpPlus.VoiceNext</RootNamespace>
-        <OutputType>Library</OutputType>
-        <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    </PropertyGroup>
-
-    <PropertyGroup>
-        <PackageId>DSharpPlus.VoiceNext</PackageId>
-        <Description>Voice implementation for DSharpPlus.</Description>
-        <PackageTags>discord, discord-api, bots, discord-bots, chat, dsharp, dsharpplus, csharp, dotnet, vb-net, fsharp, audio, voice, radio, music</PackageTags>
-    </PropertyGroup>
-
-    <ItemGroup>
-        <PackageReference Include="DSharpPlus.VoiceNext.Natives"/>
-    </ItemGroup>
-
-    <ItemGroup>
-        <ProjectReference Include="..\DSharpPlus\DSharpPlus.csproj"/>
-    </ItemGroup>
-
-</Project>
+  <PropertyGroup>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
+    <PackageId>DSharpPlus.VoiceNext</PackageId>
+    <Description>Voice implementation for DSharpPlus.</Description>
+    <PackageTags>$(PackageTags), audio, voice, radio, music</PackageTags>
+    <IsPackable>true</IsPackable>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\DSharpPlus\DSharpPlus.csproj"/>
+    <PackageReference Include="DSharpPlus.VoiceNext.Natives"/>
```

Comment:
having both project and package references in one item group defies convention; even `dotnet add` should separate them

Reply:
`dotnet add` doesn't even support project references iirc... I also fail to see why it's convention in the first place.

Reply:
`dotnet add reference`, and while i don't know, it is what all tooling in the world uses, so it's probably better to keep that way

Reply:
All the tooling probably also uses `dotnet add reference` then. And seeing as how there is no functional difference at the moment, are you *sure*  you want me to revert that change?

Reply:
VS doesn't, or at least didn't; unsure about rider. if dotnet add just works with this then *maybe,* if dotnet add creates new groups then absolutely revert it